assets/variant-list.css

```diff
@@ -16,25 +16,36 @@ variant-list .quantity__button {
 }
 
 .variant-list__container {
-  max-height: 90rem;
-  overflow-y: auto;
   padding-bottom: 4rem;
 }
 
 .variant-list__total {
-  display: flex;
-  justify-content: space-between;
   padding-top: 2rem;
   border-top: 0.1rem solid rgba(var(--color-foreground), .08);
 }
 
+.totals__product-total {
+  text-align: center;
+}
+
 @media screen and (min-width: 750px) {
   .variant-list__total {
     position: sticky;
     bottom: 0;
     z-index: 1;
     background-color: rgb(var(--color-background));
   }
+
+  .totals__product-total {
+    text-align: right;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  .variant__items {
+    max-height: 90rem;
+    overflow-y: auto;
+  }
```

Comment:
This media query goes right after the previous one (line 31). Let's combine them.